README.md

```diff
@@ -112,6 +112,12 @@ pip install -e . -r requirements/test.pip -r requirements/develop.pip
 
 > NB: the `udata.(in|pip)` files are used by the CI to stay in sync with `udata` requirements. You shouldn't need to tinker with them on your local environment, but they might be updated by the CI when you make a Pull Request.
 
+> WARNING: whenever the dependencies change on [udata](https://github.com/opendatateam/udata), the `udata.pip` needs to be manually recompiled locally:
```

Comment:
Maybe these two lines are in contradiction :p How about adding something about *temporarily* updating it manually for now?

Reply:
Ah, yes, I should learn to read. I proposed this change, what do you think?
```
> WARNING: if you experience requirements conflicts, and some dependencies were changed on [udata](https://github.com/opendatateam/udata), the `udata.pip` might need to be manually recompiled locally:
```

Reply:
Yep, seems perfect, thank you!